views/js/controller/backoffice.js

```diff
@@ -64,6 +64,9 @@ define([
             //intercept errors
             //TODO this should belongs to the Router
             $doc.ajaxError(function (event, request, settings) {
+                // Request was manually aborted, isn't a error
+                if (request.statusText === 'abort') return;
```

Comment:
```suggestion
            $doc.ajaxError(function (event, request, settings, thrownError) {
                // Request was manually aborted, isn't a error
               if(thrownError == ‘abort’ || thrownError == ‘undefined’) return;
```

It should be safer, as `statusText` is not part of the API.